diff -urN ksambaplugin-0.5/ksambaplugin-0.5/src/kcmsambaconf.cpp ksambaplugin-0.5.new/ksambaplugin-0.5/src/kcmsambaconf.cpp
--- ksambaplugin-0.5/ksambaplugin-0.5/src/kcmsambaconf.cpp	2004-01-13 11:42:24.000000000 +0100
+++ ksambaplugin-0.5.new/ksambaplugin-0.5/src/kcmsambaconf.cpp	2007-02-07 20:48:33.444378000 +0100
@@ -193,7 +193,7 @@
 
   slotSpecifySmbConf(smbFile);
 
-};
+}
 
 
 KcmSambaConf::~KcmSambaConf() {
diff -urN ksambaplugin-0.5/ksambaplugin-0.5/src/ksambapropertiesdialogplugin.cpp ksambaplugin-0.5.new/ksambaplugin-0.5/src/ksambapropertiesdialogplugin.cpp
--- ksambaplugin-0.5/ksambaplugin-0.5/src/ksambapropertiesdialogplugin.cpp	2003-12-16 17:13:45.000000000 +0100
+++ ksambaplugin-0.5.new/ksambaplugin-0.5/src/ksambapropertiesdialogplugin.cpp	2007-02-07 20:54:19.389998250 +0100
@@ -63,7 +63,7 @@
 typedef KGenericFactory<KSambaPropertiesDialogPlugin, KPropertiesDialog> KSambaPropertiesDialogPluginFactory;
 
 K_EXPORT_COMPONENT_FACTORY( ksambakonqiplugin,
-                            KSambaPropertiesDialogPluginFactory("ksambakonqiplugin") );
+                            KSambaPropertiesDialogPluginFactory("ksambakonqiplugin") )
 
 
 
